xds/internal/balancer/priority/balancer_priority.go

```diff
@@ -235,7 +235,7 @@ func (b *priorityBalancer) handleChildStateUpdate(childName string, s balancer.S
 	case connectivity.Connecting:
 		b.handlePriorityWithNewStateConnecting(child, priority, oldState)
 	default:
-		// New state is Idle, should never happen. Don't forward.
+		// New state is Shutdown, should never happen. Don't forward.
```

Comment:
> Don't forward

Will the child never remove subconns?  Shouldn't we forward the update to the child anyway?

Reply:
This is the aggregated state (with picker), from children to parent.

SubConn states are always forwarded to the children.